packages/components/addon/components/hds/table/th-sort.hbs

```diff
@@ -0,0 +1,18 @@
+{{#each @columns as |column|}}
+  <th
+    scope="col"
+    class="hds-table__th hds-table__th--sortable"
+    aria-sort={{if (eq column @sortBy) (concat @sortOrder "ending")}}
+  >
+    <button type="button" {{on "click" (fn @setSortBy column)}}>
+      {{column}}
+      {{#if (and (eq column @sortBy) (eq @sortOrder "asc"))}}
+        <FlightIcon @name="arrow-up" />
+      {{else if (and (eq column @sortBy) (eq @sortOrder "desc"))}}
+        <FlightIcon @name="arrow-down" />
+      {{else}}
+        <FlightIcon @name="swap-vertical" />
+      {{/if}}
```

Comment:
We should move all this logic in the backing class, and have a single `<FlightIcon>` declaration, with a computed `@name` value.